pallets/liquidity-proxy/src/mock.rs

```diff
@@ -925,6 +940,54 @@ impl ExtBuilder {
         self
     }
 
+    #[cfg(feature = "wip")] // DEFI-R
+    pub fn with_permissioned_xyk_pool(mut self) -> Self {
+        self = self.with_xyk_pool();
+        self.is_permissioned_xyk_pool = true;
+        self
+    }
+
+    #[cfg(feature = "wip")] // DEFI-R
+    fn prepare_asset_for_permissioned_pool(owner: &AccountId, asset_id: &AssetId) {
+        use frame_support::assert_ok;
+        System::set_block_number(1);
+        let owner_origin = RuntimeOrigin::signed(owner.clone());
+        if !ExtendedAssets::regulated_asset(asset_id) {
+            ExtendedAssets::regulate_asset(owner_origin.clone(), *asset_id)
+                .expect("Failed to regulate Asset");
+        }
+
+        ExtendedAssets::issue_sbt(
+            owner_origin.clone(),
+            AssetSymbol(b"SBT".to_vec()),
+            AssetName(b"SBT".to_vec()),
+            None,
+            None,
+            None,
+        )
+        .expect("Failed to issue SBT");
+
+        // Extract the issued SBT asset ID
+        let event = frame_system::Pallet::<Runtime>::events()
+            .pop()
+            .expect("Expected at least one event")
+            .event;
+        let sbt_asset_id = match event {
```

Comment:
You can add public function to extended assets pallet to register sbt, return asset id from this function and use it here